tests/autoyast/installation.pm

```diff
@@ -18,7 +18,8 @@
 
 use strict;
 use warnings;
-use base 'y2logsstep';
+use base 'opensusebasetest';
+use y2_logs_helper;
```

Comment:
Same as before, this changes behavior, we should use `lib/y2_installbase` here as parent, in general for most of the files that should be the only change, one parent class to another, other changes should not be required.

Reply:
fixed